src/js/map/map.js

```diff
@@ -282,14 +282,21 @@ class Map {
       this.maxZoom = 12;
     }
 
+     //set a max zoom if blockZoomToMasterMap is true to block the detailed view. By default, the max soom is 12 and zoom to MasterMap.
+     if (this.mapConfig.minMapZoom){
+      this.minZoom = this.mapConfig.minMapZoom;
+    } else {
+      this.minZoom = 4;
```

Comment:
why not use MIN_ZOOM here instead of 4?
If not, the MIN_ZOOM constant is not being used anywhere and we can delete it in consts.js.